projects/storefrontapp/server.ts

```diff
@@ -16,13 +17,14 @@ const express = require('express');
 const ssrOptions: SsrOptimizationOptions = {
   concurrency: 20,
   timeout: Number(process.env.SSR_TIMEOUT ?? 3000),
+  reuseCurrentRendering: true,
```

Comment:
To be removed?

Reply:
I'd rather have it on by default in our own setup